contexts/webkms-v1.jsonld

```diff
@@ -0,0 +1,29 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "@protected": true,
+
+    "AesKeyWrappingKey2019": "https://w3id.org/security#AesKeyWrappingKey2019",
+    "DeleteKeyOperation": "https://w3id.org/security#DeleteKeyOperation",
+    "DeriveSecretOperation": "https://w3id.org/security#DeriveSecretOperation",
+    "ExportKeyOperation": "https://w3id.org/security#ExportKeyOperation",
+    "GenerateKeyOperation": "https://w3id.org/security#GenerateKeyOperation",
+    "KmsOperation": "https://w3id.org/security#KmsOperation",
+    "RevokeKeyOperation": "https://w3id.org/security#RevokeKeyOperation",
+    "Sha256HmacKey2019": "https://w3id.org/security#Sha256HmacKey2019",
+    "SignOperation": "https://w3id.org/security#SignOperation",
+    "UnwrapKeyOperation": "https://w3id.org/security#UnwrapKeyOperation",
+    "VerifyOperation": "https://w3id.org/security#VerifyOperation",
+    "WrapKeyOperation": "https://w3id.org/security#WrapKeyOperation",
+    "invocationTarget": {
+      "@id": "https://w3id.org/security#invocationTarget", "@type": "@id"
+    },
+    "kmsModule": "https://w3id.org/security#kmsModule",
+    "publicKey": "https://w3id.org/security#publicKey",
+    "signatureValue": "https://w3id.org/security#signatureValue",
+    "unwrappedKey": "https://w3id.org/security#unwrappedKey",
+    "verifyData": "https://w3id.org/security#verifyData",
```

Comment:
We have an opportunity to type this stuff now (we could use `sec:multibase`) so its compressible with CBOR-LD. If we don't do that now, we should at least file an issue.

Reply:
K, added a multibase type.